models/stg_stripe__balance_transaction.sql

```diff
@@ -24,18 +21,25 @@ fields as (
 ),
 
 final as (
-    
     select 
         id as balance_transaction_id,
+        
+        {% if var('stripe__amount_divide', True) %}
+        amount / 100.0 as amount,
+        fee / 100.0 as fee,
+        net / 100.0 as net,
+        {% else %}
         amount,
+        fee,
+        net,
+        {% endif %}
+        
```

Comment:
Would it make more sense to make this a macro and then apply the macro to each field that requires this update throughout the package. This way if we want to make this more dynamic in the future, we'll be able to maintain that more easily?

Reply:
Great point! I made a macro called `convert_values` which contains the conditional division and also allows for aliasing in case the field also gets renamed.